acados/ocp_nlp/ocp_nlp_dynamics_disc.c

```diff
@@ -142,10 +142,45 @@ void ocp_nlp_dynamics_disc_dims_set(void *config_, void *dims_, const char *dim,
     }
     else
     {
-        assert(0 == 1);  // dimension type not available in module
+        printf("\ndimension type not available in module\n");
```

Comment:
a bit more specific: something like `("dimension type %s not available in module ocp_nlp_dynamics_disc", field)`